packages/block-library/src/missing/edit.native.js

```diff
@@ -27,13 +27,17 @@ export class UnsupportedBlockEdit extends Component {
 		const title = blockType ? blockType.settings.title : __( 'Unsupported' );
 		const titleStyle = getStylesFromColorScheme( styles.unsupportedBlockMessage, styles.unsupportedBlockMessageDark );
 
+		const subTitleStyle = getStylesFromColorScheme( styles.unsupportedBlockSubtitle, styles.unsupportedBlockSubtitleDark );
+		const subtitle = blockType ? <Text style={ subTitleStyle }>{ __( 'Unsupported' ) }</Text> : null;
```

Comment:
👋 @maxme , not sure I'm missing context, why don't we display the `Unsupported` subtitle for all unsupported blocks?

Reply:
The title for unsupported block we don't know the name is currently `Unsupported`. If we showed the subtitle, we would show `Unsupported` twice.

See the last unsupported block on this screenshot:

<img width="410" alt="Screenshot 2019-10-25 at 15 18 35" src="https://user-images.githubusercontent.com/40213/67574546-eeea3100-f73a-11e9-99a7-028dffe0c7d3.png">